start.markdown

```diff
@@ -3,7 +3,64 @@ title: Getting Started
 layout: main
 ---
 
-- [Getting O2](/git)
-- [Building O2](/building)
-- [Running O2](/running)
-- [Further material](/links)
+Get O²
+======
+
+The ALICE O² Software is hosted on [GitHub](https://github.com) under the
+[AliceO2Group](https://github.com/AliceO2Group/) organization. The core package is
+[AliceO2](https://github.com/AliceO2Group/AliceO2).
+
```

Comment:
Lets add something like:

Clone the project with
```
git clone https://github.com/AliceO2Group/AliceO2
```

Reply:
Why? We have a separate build guide, what use is that? In the build guide, we tell exactly how and where to do that, a random `git clone` out of the blue is of no practical use here IMHO.

Reply:
(I have added it in any case.)